static/js/compose.js

```diff
@@ -464,6 +467,7 @@ function validate_stream_message_mentions(stream_name) {
 
             $("#compose-send-button").prop('disabled', false);
             $("#sending-indicator").hide();
+            $(".compose_table .drafts-link").css("top", "2px");
```

Comment:
You can set css properties of `.compose_table .drafts-link` element in proper css file, instead of js files. It's good practice to keep js code and css style in separate files.